pandas/_libs/indexing.pyi

```diff
@@ -0,0 +1,10 @@
+# flake8: noqa
```

Comment:
im not sure if it helps, but indexing.pyx is one of the few cython files that _can_ be made into valid python

Reply:
Cool. Thanks.

my original plan, using http://blog.zulip.org/2016/10/13/static-types-in-python-oh-mypy/ as a suitable template for a project plan was to include the 'skeleton' stubs in the preliminary phase.

the reasoning being that if the cython files were python files, mypy would have the same information that is being made available with the skeleton stubs (i.e. without any type hints).

This was also the reason not to include the types in the stubs at this stage, to keep the preliminary phase separate from phase 1.

However, adding the stubs is increasing the mypy errors reported by check-untyped-defs, so was making no progress trying to get check-untyped-defs enabled globally.

so in order to keep things manageable, the preliminary phase needs to be split in two. I've left the stubs in this branch for now but do not intend to add any more to this branch.

Reply:
Makes sense, thanks.  Also FWIW, cython understands the py36 style variable annotations even in py35, see e.g. lib.memory_usage_of_objects